pkg/api/platformapi/allocatorapi/vacate_full_test.go

```diff
@@ -480,7 +480,10 @@ func TestVacate(t *testing.T) {
 								plan: []*models.ClusterPlanStepInfo{
 									newPlanStep("step1", "success"),
 									newPlanStep("step2", "success"),
-									newPlanStepWithDetails("last step", "error", []*models.ClusterPlanStepLogMessageInfo{{
+									newPlanStepWithDetails("step3", "error", []*models.ClusterPlanStepLogMessageInfo{{
+										Message: ec.String(planStepLogErrorMessage),
+									}}),
+									newPlanStepWithDetails("plan-completed", "error", []*models.ClusterPlanStepLogMessageInfo{{
```

Comment:
Is it safe to say `plan-completed` is the only step ID we care about in terms of errors from the product's perspective?  It's more edification for myself, as I'm unsure if errors for other step IDs get bubbled up into the plan completed step or if we're just "ignoring" everything, but the plan completed step, since it contains the only relevant errors.

Reply:
So the product does bubble up the errors to the plan-completed step when they're relevant.

 Since the kntrroduction of the plan step retries to make plans more resilient, the previous behavior which was erroring should any plan step be error, was causing A Lot of error duplications on plans.